interfaces/builtin/fwupd.go

```diff
@@ -388,9 +388,16 @@ func (iface *fwupdInterface) UDevPermanentSlot(spec *udev.Specification, slot *s
 	if !implicitSystemPermanentSlot(slot) {
 		spec.TagDevice(`KERNEL=="drm_dp_aux[0-9]*"`)
 		spec.TagDevice(`KERNEL=="gpiochip[0-9]*"`)
-		spec.TagDevice(`KERNEL=="tpm[0-9]*"`)
```

Comment:
Why are these removed?

Reply:
the list was sorted, tpm is few lines down

Reply:
Ah okay